sample/src/main/java/vehicles/AbstractVehicle.java

```diff
@@ -0,0 +1,40 @@
+package vehicles;
+
+import people.Person;
+
+import java.time.LocalDate;
+
+/**
+ * An abstract class for implementing different kinds of {@link Vehicle}.
+ *
+ * @author http://twitter.com/manoelcampos[Manoel Campos da Silva Filho]
+ */
+public class AbstractVehicle implements Vehicle {
+    private int manufacturingYear;
```

Comment:
To avoid redundant documentation among the field and its getter and setter, I usually don't comment the field. Instead, I just use a @see tag to the getter (an asciidoc link in this case).
What do you think?